linux/org.raspberrypi.rpi-imager.metainfo.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--Copyright 2020 Flathub maintainers-->
+<component type="desktop">
+  <id>org.raspberrypi.rpi-imager</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>Apache-2.0</project_license>
+  <name>Raspberry Pi Imager</name>
+  <summary>Raspberry Pi imaging utility</summary>
+  <description>
+    <p>
+      Raspberry Pi Imager downloads a .JSON file from our website with a list of all current download options, ensuring you are always installing the most up-to-date version.
+    </p>
+    <p>
+      Once you’ve selected an operating system from the available options, the utility reads the relevant file directly from our website and writes it straight to the SD card. This speeds up the process quite considerably compared to the standard process of reading it from the website, writing it to a file on your hard drive, and then, as a separate step, reading it back from the hard drive and writing it to the SD card.
+    </p>
+    <p>
+      During this process, Raspberry Pi Imager also caches the downloaded operating system image – that is to say, it saves a local copy on your computer, so you can program additional SD cards without having to download the file again.
+    </p>
+  </description>
+  <launchable type="desktop-id">rpi-imager.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://www.raspberrypi.org/app/uploads/2020/03/RPI_intro-e1583228263677.png</image>
```

Comment:
@XECDesign are these URLs guaranteed to continue to exist, or should they be changed to something else?

Reply:
I'm not sure, it's not anything I'm familiar with.

Reply:
It's not that much of an issue if these snapshots were to move at a future time. You'd just have to update the metadata when it happens.

Reply:
@XECDesign Would it make sense to copy these screenshots to http://downloads.raspberrypi.org/imager/ ?

Reply:
@lurch done

Reply:
@lurch updated